.github/workflows/publish.yaml

```diff
@@ -22,25 +20,22 @@ jobs:
           profile: minimal
           toolchain: stable
           override: true
-      - uses: actions-rs/cargo@v1
+      - run: rustup component add rustfmt
```

Comment:
Sorry if I'm wrong, but is this step absolutely necessary?

Reply:
Done in https://github.com/tchataigner/mock-crate/pull/4/commits/efb62c19bc4e95067d96c785cf0e96f513af0bfc